apps/train_loc/lib/train_loc/encoder/vehicle_positions_enhanced.ex

```diff
@@ -59,6 +94,8 @@ defmodule TrainLoc.Encoder.VehiclePositionsEnhanced do
     %{start_date: start_date(vehicle.timestamp)}
   end
 
+  @spec entity_vehicle(Vehicle.t()) ::
```

Comment:
suggestion: since you're re-using this from the `entity` spec, maybe pull it out and give it a name to use here?